examples/seismic/viscoacoustic/wavesolver.py

```diff
@@ -32,6 +32,9 @@ def __init__(self, model, geometry, space_order=4, kernel='blanch_symes', **kwar
 
         self.space_order = space_order
         self.dt = self.model.critical_dt
+        if kernel not in {'blanch_symes', 'ren', 'deng_mcmechan'}:
```

Comment:
no...this should be made clean by having viscoacoustic use its own arg for kernel

Reply:
Out of seismic_args I guess?